src/PerformanceSensitiveAnalyzers/UnitTests/ExplicitAllocationAnalyzerTests.cs

```diff
@@ -250,11 +179,11 @@ public class MyClass
     public void Foo() 
     {
         System.ValueType box = new S();
+        S noBox = new S();
```

Comment:
💭 Would it make sense to have a separate type for this?

Reply:
Makes sense to make the tests more granular. I will pull this one out and also pull apart the ExplicitAllocation_ObjectInitializer test.